pkgs/development/haskell-modules/hackage-packages.nix

```diff
@@ -247824,7 +247824,7 @@ self: {
          base bytestring cabal-test-bin hspec hspec-test-sandbox process
          shakespeare test-sandbox text unix
        ];
-       description = "Lightweight development enviroments using test-sandbox";
+       description = "Lightweight development environments using test-sandbox";
```

Comment:
This file is automatically updated from Haskell packages on http://hackage.haskell.org/.  Even if you update this, it will just get overwritten on the next automatic update.

Reply:
Ah cool, thanks, I wasn't giving too much attention. Removing this one from PR.